mailpoet/lib/Automation/Engine/Workflows/Workflow.php

```diff
@@ -55,6 +56,10 @@ public function getStatus(): string {
     return $this->status;
   }
 
+  public function setDraft(): void {
+    $this->status = self::STATUS_DRAFT;
+  }
```

Comment:
There's probably a better way of handling this than adding a public method like this. Maybe there should be a constructor argument for the status instead? Not sure what feels right.

Reply:
I think now `draft` should be the default. Or do we set it automatically to `inactive` when it's already valid? Not sure if that's needed. We may simply have only drafts and only allow activation when the workflow is valid. In that sense, we'd just rename "inactive" to "draft".

What do you think? Not sure about this.

Reply:
I definitely agree about changing the default to be `draft, but I think it will still be useful to have the 3 states.

Users will certainly want the ability to turn a specific automation on or off, so I like the idea of having `inactive` for that.

Some users will probably have a lot of workflows, and it may be helpful for them to be able to view just the drafts instead of seeing drafts and inactives together, for example if they have multiple that are a work in progress and they want to find all of those and finish them.

So perhaps a `draft` can only be manually transitioned into `active`, and then it can go back and forth between `active` and `inactive`. If we did that it might almost make sense to rename `inactive` to `paused`.

Then there's the question of what should the status be if a workflow becomes invalid somehow (e.g. a user deletes the segment the trigger is based on). Should the workflow be changed back to a draft? Should there be an `invalid` status? Maybe a question for another ticket.

For now I like the idea of keeping the 3 statuses and changing the default to `draft` but I'm open to other ideas.